javers-core/src/test/groovy/org/javers/core/diff/appenders/OptionalChangeAppenderTest.groovy

```diff
@@ -9,6 +9,8 @@ import spock.lang.Unroll
 import static java.util.Optional.*
```

Comment:
@mwesolowski as a rule of thumb yes, you are right but there are exceptions
JaVers sits in the middle between Java7 and Java8

It's quite complicted, i have to clarify this in Contributor Guidlines ...
Whole /src should be written in Java7, except of packages like `package org.javers.java8support`
In /test Java8 is allowed

So in this case Java8 is OK

Big Picture is: we want JaVers to be runable on JDK7 but also want to support new Java type (`Java8AddOns`)